Legacy/offline/debugging/rbsm_lib.py

```diff
@@ -7,7 +7,7 @@ class RBSerialMessage:
 
     def __init__(self, device_path):
         self.RBSM_FOOTER = "\n" # 0x0A
-        self.RBSM_BAUD_RATE = 76800
+        self.RBSM_BAUD_RATE = 57600
```

Comment:
We should change it such that the baud rate is in RBSM_Config.  We wasted time today debugging why the terminal wasn't connecting, and likewise high level has experienced similar confusion.